src/ui/chat/ChatMessageBar.js

```diff
@@ -6,6 +6,7 @@ import type {Conversation} from '../../model';
 type Props = {
   conversation: ?Conversation,
   onSubmit: string => any;
+  onDraft: string => any;
```

Comment:
Given that this is not used for games, I'd make it optional: `onDraft?: string => any`